### PR TITLE
Refactor LTR docs

### PR DESCRIPTION
**Pull request by lintool**

### CODE REVIEW

File: pyserini/search/lucene/ltr/__main__.py [hunk 1]
```diff
@@ -206,27 +207,39 @@ def output(file, dev_data, format):
...
+                    output_file.write(f"{qid}\t{docid}\t{rank}\n")
+                rank += 1
     score_tie = f'score_tie occurs {score_tie_counter} times in {len(score_tie_query)} queries'
     print(score_tie)
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(description='Learning to rank reranking')
     parser.add_argument('--input', required=True)
-    parser.add_argument('--reranking-top', type=int, default=1000)
+    parser.add_argument('--reranking-top', type=int, default=10000)
```
Comment (lintool) [inline pyserini/search/lucene/ltr/__main__.py:233]: Can you use the same args as the main searcher?
https://github.com/castorini/pyserini/blob/master/pyserini/search/lucene/__main__.py#L112

File: scripts/ltr_msmarco/generate_document_score_withmaxP.py [hunk 2]
```diff
@@ -26,4 +26,4 @@
             #docid_score.sort(key=lambda x:x[1], reverse=True)
```
Comment (lintool) [inline scripts/ltr_msmarco/generate_document_score_withmaxP.py:26]: Please delete dead code... throughout?

File: pyserini/search/lucene/ltr/_search_msmarco.py [hunk 3]
```diff
@@ -34,14 +37,20 @@
...
+        index_directory = os.path.join(get_cache_home(), 'indexes')
+        index_path = os.path.join(index_directory, 'index-msmarco-passage-ltr-20210519-e25e33f.a5de642c268ac1ed5892c069bdc29ae3')
+        self.fe = FeatureExtractor(index_path, max(multiprocessing.cpu_count()//2, 1))
+        self.index_reader = IndexReader.from_prebuilt_index(index)
+        self.data = data
+
     
     def add_fe(self):
         #self.fe.add(RunList('collections/msmarco-ltr-passage/run.monot5.run_list.whole.trec','t5'))
+        #self.fe.add(RunList('../bert.whole.doc.trec','bert'))
```
Comment (lintool) [inline pyserini/search/lucene/ltr/_search_msmarco.py:53]: Kill dead code?

File: integrations/sparse/test_ltr_msmarco_document.py [hunk 4]
```diff
@@ -34,8 +34,7 @@ def test_reranking(self):
         outp_tsv = 'run.ltr.msmarco-pass-doc.test.tsv'
         #Download prebuilt index
         #retrieve candidate
-        LuceneSearcher.from_prebuilt_index('msmarco-doc-per-passage-ltr')
-        os.system(f'python -m pyserini.search --topics msmarco-doc-dev  --index ~/.cache/pyserini/indexes/index-msmarco-doc-per-passage-ltr-20211031-33e4151.bd60e89041b4ebbabc4bf0cfac608a87/ --output ltr_test/{inp} --bm25 --output-format trec --hits 10000')
+        os.system(f'python -m pyserini.search --topics msmarco-doc-dev  --index msmarco-doc-per-passage-ltr --output ltr_test/{inp} --bm25 --output-format trec --hits 10000')
```
Comment (yuki617) [inline integrations/sparse/test_ltr_msmarco_document.py:37]: Should pyserini.search be changed into pyserini.search.lucene?

**Pull request merged**